index.html

```diff
@@ -41,32 +49,40 @@ <h2 id="description-text">I'm a Software Developer</h2>
                          
                 </section>
             </section>  
-        </article> 
+            
+        
+    </article> 
         <article id="work-section">
                 <section class="recent-work">
-                    <h2 id="recent-work-title">My Recent Works</h2>
-                    <hr>
+                    <div class="title-recent-word"><h2 id="recent-work-title">My Recent Works</h2>
+                    <hr></div>
                     <img id="img-project" src="images/Img-Placeholder.png" alt="images-work">
+                    <div class="multipost">
+                    <div class="big-image"></div>
+                    <div>
                     <h3 id="project-description-title">Multi-Post Stories</h3>
```

Comment:
- [ ] Please consider keeping this design in desktop design mode for a screen size higher than 780px. This can be done by keeping the image and the description on the same row (as on the figma design) and reduce progressively the image size when the screen is reduced. Please always remember replicating exact Figma templates gives confidence to recruiters in your ability to replicate business requirements as regards design 😄

**Your project design** There is still a lot of space so the image and description should stay in one row
![image](https://user-images.githubusercontent.com/74411135/229297076-3bf01068-7967-4204-b674-0ed94fd7ab09.png)

**Figma design**
![image](https://user-images.githubusercontent.com/74411135/229297111-e4c31219-757e-4eee-850e-64e0fda9fdb5.png)